contracts/priv/unity.yml

```diff
@@ -200,6 +200,31 @@ paths:
         default:
           description: Unexpected error
           $ref: '#/components/responses/ServerError'
+  /allowances/orgs/create:
+    get:
+      operationId: GetAllowancesOrgsCreate
+      tags:
+        - Allowances
+        - Organizations
+      summary: Return whether a new Organization can be created.
+      parameters:
+        - $ref: '#/components/parameters/TraceSpan'
+      responses:
+        '200':
+          description: The Allowance
+          content:
+            application/json:
+              schema:
+                properties:
+                  allowed:
```

Comment:
Should the name for this property be generic (`allowed`) or should it be something more specific (like `canCreateOrg`)? Either seems okay to me, but since the response has two booleans, we're kind of providing two pieces of information about what's 'allowed'.

Reply:
The schema name in the source is `Allowance`, so it's intended to be generic, because this return payload can be reused for any allowance query. Or at least, that's my thinking :-).

The other boolean indicates whether an upgrade might affect the answer to the question of whether something is allowed or not.

Reply:
Makes sense to me. 👍